src/odemis/acq/test/stream_test.py

```diff
@@ -564,13 +569,46 @@ def test_fluo(self):
             exp_intens[i] = min(self.light.power.range[1][i], s1.power.value)
             self.assertEqual(self.light.power.value, exp_intens)
 
-        time.sleep(2)
+        self._image_received.wait(2)
         s1.is_active.value = False
 
         self.assertFalse(self._image is None, "No image received after 2s")
 
+        # Check the actual FoV matched the expected one
+        im_bbox = img.getBoundingBox(self._image)
+        im_fov = im_bbox[2] - im_bbox[0], im_bbox[3] - im_bbox[1]
+        self.assertAlmostEqual(guessed_fov, im_fov)
+
+    def test_sem(self):
+
+        sems = stream.SEMStream("test sem", self.sed, self.sed.data, self.ebeam,
+                                emtvas={"scale"})
+        self.ebeam.dwellTime.value = self.ebeam.dwellTime.range[0]
+
+        # Try to trick the guessFoV by setting a different scale on the e-beam than in the local VA
+        self.ebeam.scale.value = (1, 1)
+        sems.emtScale.value = (3, 3)
```

Comment:
Yes, that's right. The emtScale is a copy of the orignal scale, but allows the stream to hold onto that setting until it's played. It's used so that you can have multiple streams, with different settings (aka local VAs), and whenever you play/acquire with one stream, the settings are used.